core/src/main/scala/org/apache/spark/ui/exec/ExecutorsPage.scala

```diff
@@ -159,10 +181,14 @@ private[ui] class ExecutorsPage(
       }
       {
         if (threadDumpEnabled) {
```

Comment:
when threadDump is enable, I think "<td> </td>" is better than Seq.empty for dead executor, because active executor and dead executor are in the one table. It is better for table alignment.